evalml/pipelines/classification/baseline_binary.py

```diff
@@ -3,12 +3,25 @@
 
 class BaselineBinaryPipeline(BinaryClassificationPipeline):
     """Baseline Pipeline for binary classification."""
-    custom_name = "Baseline Classification Pipeline"
+    custom_name = "Baseline Binary Classification Pipeline"
     component_graph = ["Baseline Classifier"]
+    custom_hyperparameters = None
 
+    def __init__(self, parameters, random_seed=0):
+        return super().__init__(self.component_graph,
```

Comment:
Long-term, I think it's okay to keep the classes as is since we use them in AutoMLSearch, and having the class makes it clear what the pipeline is for! :)